scrapy_zyte_smartproxy/middleware.py

```diff
@@ -214,6 +224,19 @@ def process_request(self, request, spider):
         if self._is_enabled_for_request(request):
             if 'proxy' not in request.meta:
                 request.meta['proxy'] = self._auth_url
+            elif (
+                request.meta['proxy'] == self._authless_url
+                and b"Proxy-Authorization" not in request.headers
+            ):
+                logger.warning(
```

Comment:
@Gallaecio Would make sense to be able to silent this warning? I'm migrating Crawlera to Zyte API Proxy mode and have a bunch spiders that pass `meta=response.meta` that end up triggering this warning a bunch of times.

Reply:
`meta=response.meta` is a bad practice. You should only copy the specific meta keys you want to pass on, or use something like https://github.com/heylouiz/scrapy-sticky-meta-params/.

Reply:
Got it! Thank you I will refactor my spiders to follow better practice.